src/rules/no_unreachable.rs

```diff
@@ -35,6 +35,66 @@ impl LintRule for NoUnreachable {
       ProgramRef::Script(ref s) => visitor.visit_script(s, &DUMMY_NODE),
     }
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows the unreachable code after the control flow statements.
+
+Because the control flow statements(`return`,`throw`, `break` and `continue`) unconditionally exit a block of code, any statements after them cannot be executed.
```

Comment:
```suggestion
Because the control flow statements (`return`, `throw`, `break` and `continue`) unconditionally exit a block of code, any statements after them cannot be executed.
```